tpm2_pytss/tsskey.py

```diff
@@ -0,0 +1,82 @@
+"""
+SPDX-License-Identifier: BSD-3
+"""
+
+from ._libtpm2_pytss import lib
+from .types import *
+from asn1crypto.core import ObjectIdentifier
+
+
+parent_rsa_template = TPMT_PUBLIC() # FIXME, use template from tpm2-openssl
+parent_ecc_template = TPMT_PUBLIC() # FIXME, use template from tpm2-openssl
+
+loadablekey_oid = ObjectIdentifier("2.23.133.10.1.3")
+
+class TSSPrivKey(object):
+    def __init__(self, private, public, emptyAuth=True, parent=lib.TPM2_RH_OWNER):
+        self.private = private
+        self.public = public
+        self.emptyAuth = emptyAuth
+        self.parent = parent
+
+    @property
+    def private(self):
+        return self.private
+
+    @property
+    def public(self):
+        return self.public
+
+    @property
+    def emptyAuth(self):
+        if self.emptyAuth:
+            return True
+        return False
```

Comment:
isn't this just a Bool, can't we return it directly?

Reply:
it's probably a bool, but it's not certain, so if someone passed emptyAuth=1 to init it will not be a bool.
I can move the check to init, which is probably better

Reply:
```python
return bool(self.emptyAuth)
```